components/apimgt/org.wso2.carbon.apimgt.rest.api.publisher/src/gen/java/org/wso2/carbon/apimgt/rest/api/publisher/ApisApi.java

```diff
@@ -40,6 +40,7 @@
    private final ApisApiService delegate = ApisApiServiceFactory.getApisApi();
 
     @DELETE
+    @OPTIONS
```

Comment:
Although we have added these temporary, we need to update the code-gen for adding this option method for all the resources.

Reply:
@RenukaFernando Please note.